.github/workflows/dotnet-release.yml

```diff
@@ -0,0 +1,43 @@
+name: .NET Release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  build-lin:
+    name: Build and test Linux
+    uses: ./.github/workflows/build-test-linux.yml
+      
+  build-win:
+    name: Build and test on Windows
+    uses: ./.github/workflows/build-test-win.yml
+      
+  deploy:
+    needs: [build-lin, build-win]
```

Comment:
Why does the deployment need both Linux and Windows build binaries?

Reply:
I wanted to make sure that build and deployment run build and test again before publishing. In our projects it is standard practise to have CI runs with tests and still perform them again in the release pipeline.

Reply:
Fair enough. However, the deployment pipeline doesn't actually use the tested binaries.

Reply:
No it does not. Uploading and downloading artifacts did not seem worth the effort. I only use it as a condition.